lib/kubernetes-deploy/runner.rb

```diff
@@ -69,39 +69,24 @@ class Runner
 
     def self.with_friendly_errors
       yield
-    rescue FatalDeploymentError => error
-      KubernetesDeploy.logger.fatal <<-MSG
```

Comment:
I moved this logging into a rescue on `run` (that re-raises, so we still land here) so that the output if you're tophatting via a test (which does not use `with_friendly_errors`) is consistent with what you'd see by running the executable (which does).